Extend/Payment_Main.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+use \OxidEsales\Eshop\Core\Registry;
+
+/**
+ * Class Payment_Main
+ *
+ */
+class Payment_Main extends Payment_Main_parent
+{
+
+    /**
+     * Executes parent::render(), creates a currency array
+     *
+     * @return string
+     *
+     */
+    public function render()
+    {
+        parent::render();
```

Comment:
The return value of this is `payment_main.tpl`. You could save this and return it below so you don't have to hardcode it.